models/load_pearl_of_great_price.py

```diff
@@ -1,12 +1,53 @@
 """Load Pearl of Great Price."""
 
+import re
+
 from bs4 import BeautifulSoup
 from langchain.schema.document import Document
 
 from models.load_utils import clean
 from models.load_utils import to_markdown
 
 
+def remove_links(markdown_content: str) -> str:
+    """Remove links from the document."""
+    pattern = r"\[(.*?)\]\((.*?)\)"
+    return re.sub(pattern, "", markdown_content)
+
+
+def remove_lines_starting_with_pipe(text: str) -> str:
```

Comment:
Lines that start with a pipe indicate a table row. I think we need to keep these. Some table rows contain good text.